bpf/process/bpf_fork.c

```diff
@@ -23,14 +23,15 @@ BPF_KPROBE(event_wake_up_new_task, struct task_struct *task)
 	struct execve_map_value *curr, *parent;
 	struct task_struct *current_task;
 	struct tetragon_conf *config;
-	u32 pid = 0, ppid = 0, error_flags = 0;
+	u32 tgid = 0, ctgid = 0, error_flags = 0;
```

Comment:
hum, not sure ctgid is better name, ppid makes more sense to me

Reply:
ok will change back, thanks